runtime/lua/vim/lsp/util.lua

```diff
@@ -743,6 +743,20 @@ local function bufwinid(bufnr)
   end
 end
 
+--- Get list of buffers for a directory
+---@private
+local function get_directory_buffers(path)
```

Comment:
nit (to avoid proliferation in the future)
```suggestion
local function get_dir_bufs(path)
```